.rubocop_todo.yml

```diff
@@ -971,6 +971,7 @@ Style/PerlBackrefs:
 Style/PredicateName:
   Exclude:
     - 'spec/**/*'
+    - 'src/api/app/controllers/webui/kiwi/images_controller.rb'
```

Comment:
Just to clarify: `Style/PredicateName` is for checking if a method starts with `is_` if a question mark is used at the end of a method, right? We had this discussion for the Pundit stuff and while I was coding, people said (I think it was @mdeniz) that I should remove the `has_` in front of a method name that ends with a `?`. And I think the ending `?` speaks for it self.